src/qml/QFieldLocalDataPickerScreen.qml

```diff
@@ -364,14 +364,14 @@ Page {
       title: qsTr('Item Actions')
 
       width: {
-        var result = 0;
-        var padding = 0;
-        for (var i = 0; i < count; ++i) {
-          var item = itemAt(i);
-          result = Math.max(item.contentItem.implicitWidth, result);
-          padding = Math.max(item.padding, padding);
-        }
-        return Math.min( result + padding * 2,mainWindow.width - 20);
+          var result = 50;
+          var padding = 0;
+          for (var i = 0; i < count; ++i) {
+              var item = itemAt(i);
+              result = Math.max(item.contentItem.implicitWidth, result);
+              padding = Math.max(item.leftPadding + item.rightPadding, padding);
+          }
+          return mainWindow.width > 0 ? Math.min(result + padding * 2, mainWindow.width - 20) : result + padding;
```

Comment:
I guess we cannot do much not to repeat this 10 times in the codebase. I remember I tried once in the distant past.

Reply:
@suricactus , yeah, unfortunately.